client/src/js/agreement/CreateAgreementContainer.jsx

```diff
@@ -41,6 +41,7 @@ function CreateAgreementContainer () {
                   <li>I'm making contributions of code I've created in my free time (not under contract),
                     and I hold the copyright on my contributions.
                   </li>
+                  <br></br><br></br><br></br>
```

Comment:
Adding new lines will only work on some screen resolution. The `<Grid item xs={12} md={6}>` items are the same height already, so I would look into making the card full height via `css`
![Screen Shot 2021-07-15 at 4 00 01 PM](https://user-images.githubusercontent.com/3929466/125868273-8787ac9f-cdef-42cb-ab6d-76a843452966.png)

Reply:
For example: https://stackoverflow.com/questions/55824260/same-height-cards-in-material-ui